resources/assets/less/bem/ranking-page-header.less

```diff
@@ -37,4 +39,27 @@
     color: @yellow;
     text-transform: capitalize;
   }
+
+  &__flag {
+    .link-plain();
+    &:hover {
+      .flag-country {
+        opacity: 0.5;
+      }
+      .@{top}__flag-overlay {
+        display: block;
+      }
+    }
+  }
+
+  &__flag-overlay {
+    display: none;
+    pointer-events: none;
+    font-size: 20px;
+    position: absolute;
+    left: 2px;
+    top: 10px;
```

Comment:
this just so weird. Make `__flag` an inline flex and turn this into actual overlay (with background color etc).